ivadomed/training.py

```diff
@@ -458,23 +461,24 @@ def get_metadata(metadata, model_params):
                 for k in range(len(metadata))]
 
 
-def store_film_params(gammas, betas, contrasts, metadata, model, film_layers, depth):
+def store_film_params(gammas, betas, metadata_values, metadata, model, film_layers, depth, film_metadata):
     """Store FiLM params.
 
     Args:
         gammas (dict):
         betas (dict):
-        contrasts (list): list of the batch sample's contrasts (eg T2w, T1w)
+        metadata_values (list): list of the batch sample's metadata_values (eg T2w, T1w)
```

Comment:
Not sure to understand the difference between `metadata_values` and `film_metadata`: could you please help me? :-)

Reply:
Oh, is `film_metadata` a dict key? so that we can select a specific metadata. If so, all good! Maybe make it clearer in docstring? Thanksss

Reply:
film metadata is the metadata used for film e.g. contrast, tumor_type, task while metadata values is the value e.g. T1w, astrocytoma or spleen segmentation

Reply:
clarified here: 96638cf